gulp/tasks/copy.js

```diff
@@ -36,10 +38,30 @@ gulp.task( 'copy:vendorjs', function() {
     } ) );
 } );
 
+gulp.task( 'copy:usage', function() {
+  return gulp.src( config.usage.src )
+    .pipe( $.changed( config.usage.dest ) )
+    .pipe( $.tap( function ( file, t ) {
+      fileName = path.dirname( file.path )
+        .split( path.sep )
+        .pop();
+    } ) )
+    .pipe( $.rename( function ( path ) {
+      path.basename = fileName;
+      path.extname = '.md'
+    } ) )
+    .on( 'error', handleErrors )
+    .pipe( gulp.dest( config.usage.dest ) )
+    .pipe( browserSync.reload( {
+      stream: true
+    } ) );
+} );
+
 gulp.task( 'copy',
   [
     'copy:files',
     'copy:icons',
-    'copy:vendorjs'
+    'copy:vendorjs',
+    'copy:usage'
```

Comment:
Trivial: Could add a comma here to prevent the need to modify this line in the future when adding something else to the list.